plugins/grpc/go/v1.5.0/separate-package.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/cmd/protoc-gen-go-grpc/grpc.go b/cmd/protoc-gen-go-grpc/grpc.go
+index abc21602..7d9dd638 100644
+--- a/cmd/protoc-gen-go-grpc/grpc.go
++++ b/cmd/protoc-gen-go-grpc/grpc.go
+@@ -20,6 +20,7 @@ package main
+ 
+ import (
+ 	"fmt"
++	"path"
+ 	"strconv"
+ 	"strings"
+ 
+@@ -136,8 +137,27 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) *protogen.Generated
```

Comment:
This part of the patch didn't change, correct?

Reply:
Nope it did not change.

```
...
< index abc21602..7d9dd638 100644
---
> index 9088bc2a..290d012e 100644
13c13
< @@ -136,8 +137,27 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) *protogen.Generated
---
> @@ -132,8 +133,27 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) *protogen.Generated
44c44
< index 220a2a28..69971872 100644
---
> index 0cb6a021..69055d3e 100644
...
```

Reply:
```
diff --git a/cmd/protoc-gen-go-grpc/grpc.go b/cmd/protoc-gen-go-grpc/grpc.go
index 9088bc2a..abc21602 100644
--- a/cmd/protoc-gen-go-grpc/grpc.go
+++ b/cmd/protoc-gen-go-grpc/grpc.go
@@ -84,9 +84,12 @@ func (serviceGenerateHelper) generateUnimplementedServerType(gen
...
 
@@ -264,7 +268,7 @@ func genService(gen *protogen.Plugin, file *protogen.File, g 
...
```

This function is around line 134